pkg/apis/testing/v1alpha1/testsuite_types.go

```diff
@@ -0,0 +1,150 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// TestSuiteSpec defines the desired state of ClusterTestSuite
+type TestSuiteSpec struct {
```

Comment:
**Suggestion:**

For me, it will be more readable when you will change the order of the struct definition. I'm used to that the main CRD definition is always at the top thanks to that in an easy way I can jump to status/spec or other fields under the CRD. Currently, all projects that I know are structured in this way:

- [Kubernetes](https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/apps/types.go#L26-L48)
- [ServiceCatalog](https://github.com/kubernetes-incubator/service-catalog/blob/master/pkg/apis/servicecatalog/v1beta1/types.go#L28-L46)
- [and Kyma also](https://github.com/kyma-project/kyma/blob/master/components/idppreset/pkg/apis/authentication/v1alpha1/types.go#L8-L17)